### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by HITGIF**
- Add gitter chat badge
- Add contents section
- Add additional learning section
- Update setup and running procedure
- Update contribution guideline
- Fix grammar & command line format errors

### CODE REVIEW

**Comment by bhavyaagg** [comment]
@HITGIF Please resolve the conflicts

**Comment by HITGIF** [comment]
Done =)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Certifier Project [![Gitter chat](https://badges.gitter.im/gitterHQ/services.png)](https://gitter.im/jboss-outreach)
```
Comment (aashutoshrathi) [inline README.md]: Remove Gitter Badge from Heading

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
 
 
+[![Gitter chat](https://badges.gitter.im/gitterHQ/services.png)](https://gitter.im/jboss-outreach)
```
Comment (aashutoshrathi) [inline README.md]: Use `.svg` badges, `.png` gets pixelated.

File: README.md [hunk 3]
```diff
@@ -32,104 +43,94 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
 
 ###### Alternatively
 
-* You can also download the [.zip](https://github.com/jboss-outreach/certifier/archive/master.zip) file.
+## <a id="setup"></a> Setting up the project
+
+**If you want to just download the app**
+
+- Just download [.zip](https://github.com/jboss-outreach/certifier/archive/master.zip) file.
```
Comment (aashutoshrathi) [inline README.md]: `.zip` --> `zip`

File: README.md [hunk 4]
```diff
@@ -32,104 +43,94 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
-			$ git push origin 
+**Step 3: Commiting and pushing the changes:**
 
+*Commit* the changes and *push* the commit to your forked repository on GitHub by:
+```sh
+$ git add FILENAME.EXTENSION    # for including inidividual files
+$ git add .     # for including all files at once
+```
+```sh
+$ git commit -m"Your commit message here"
```
Comment (aashutoshrathi) [inline README.md]: Add a whitespace after `-m`

File: README.md [hunk 5]
```diff
@@ -32,104 +43,94 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
-**Step 5: Ensuring code quality** 
+**Step 5: Ensuring code quality**
 
 Once a PR has been created, check if it can be merged without any issues or conflicts. If there are any issues, repeat from **Step 2** and try to resolve them. Wait for a reviewer to cross check your changes, and then merge your changes.
 
 * Additional Reference regarding clone, fork and editing a repository [**here**](https://egghead.io/lessons/javascript-how-to-fork-and-clone-a-github-repository).
 
-## <a id = "help"> </a>Help/Reference
+```
+Keep Contributing to open source! =)
```
Comment (aashutoshrathi) [inline README.md]: If you want to use smile emoji better use `:smile:`

File: README.md [hunk 6]
```diff
@@ -32,104 +43,94 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
+```sh
+$ git commit -m"Your commit message here"
+$ git push origin
+```
 If stuck at this point, refer [here](https://readwrite.com/2013/10/02/github-for-beginners-part-2/)
 
-**Step 4: Sending a Pull Request (PR):** 
+**Step 4: Sending a Pull Request (PR):**
 
 Once you are done coding the changes, commit the files and create a [*PR*](https://help.github.com/articles/about-pull-requests/). Click on "Compare across forks" when creating the PR, and select the master branch of this repo as the base. Set the head to your branch on your fork. Click on the button "Create Pull Request". Give your PR a meaningful title and a brief message explaining the purpose of your commits.
```
Comment (aashutoshrathi) [inline README.md]: `Set the head to your branch on your fork` --> `Set the head to your branch on your committed onto`

File: README.md [hunk 7]
```diff
@@ -32,104 +43,94 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
 
-**Step 5: Ensuring code quality** 
+**Step 5: Ensuring code quality**
 
 Once a PR has been created, check if it can be merged without any issues or conflicts. If there are any issues, repeat from **Step 2** and try to resolve them. Wait for a reviewer to cross check your changes, and then merge your changes.
```
Comment (aashutoshrathi) [inline README.md]: You can give a link to `Step 2`

**Comment by HITGIF** [comment]
Done =)

File: README.md [hunk 8]
```diff
@@ -72,64 +76,66 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
+Once you are done coding the changes, commit the files and create a [*PR*](https://help.github.com/articles/about-pull-requests/). Click on "Compare across forks" when creating the PR, and select the master branch of this repo as the base. Set the head to your branch that your committed onto. Click on the button "Create Pull Request". Give your PR a meaningful title and a brief message explaining the purpose of your commits.
 
-**Step 5: Ensuring code quality** 
+**Step 5: Ensuring code quality**
 
-Once a PR has been created, check if it can be merged without any issues or conflicts. If there are any issues, repeat from **Step 2** and try to resolve them. Wait for a reviewer to cross check your changes, and then merge your changes.
+Once a PR has been created, check if it can be merged without any issues or conflicts. If there are any issues, repeat from [***Step 2***](#step2) and try to resolve them. Wait for a reviewer to cross check your changes, and then merge your changes.
```
Comment (aashutoshrathi) [inline README.md]: `and then merge your changes.` --> `and let them merge your changes.`

File: README.md [hunk 9]
```diff
@@ -1,47 +1,51 @@
...
 
-## Dependencies
-* [Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 8 (Dont use Java 9) 
-* [Gradle 4.0.0](https://gradle.org/releases/) or higher
+- Just download [zip](https://github.com/jboss-outreach/certifier/archive/master.zip) file.
+
+**If you want to contribute to the project**
 
-### Initial Setup
+1. **Fork** ![](https://camo.githubusercontent.com/ba881041f4d7c96895e8466eece0c43317919f56/68747470733a2f2f696d6167652e6962622e636f2f667953745a6d2f666f726b2e706e67) the repository by clicking on the Fork icon at the top right corner of this page.
```
Comment (aashutoshrathi) [inline README.md]: Remove the fork image, it looking ugly.

**Comment by HITGIF** [comment]
Done @aashutoshrathi

**Comment by HITGIF** [comment]
Approve the [task](https://codein.withgoogle.com/dashboard/task-instances/6438516651196416/) please if it's ok :)

File: README.md (hunk 9 above)
Comment (aashutoshrathi) [reply README.md]: Please do this

**Comment by aashutoshrathi** [comment]
Check my review comments, do the changes I requested first, then ask for any kind of approval please

**Comment by HITGIF** [comment]
Oops sorry, something got reset while I was trying to squash the commits.
All done now :)

File: README.md [hunk 10]
```diff
@@ -72,64 +76,66 @@ This project generates a certificate. It runs on Vert.x, and is built on the Gra
...
+
+## <a id = "learning"> </a> Additional Learning
 
-You can have a look this [**web page**](https://egghead.io/lessons/javascript-how-to-fork-and-clone-a-github-repository) to know how to Clone, Fork and edit the project on a local respository. 
-Keep contributing!
+- [Git Handbook](https://guides.github.com/introduction/git-handbook)
+- [Git Introduction](https://guides.github.com/introduction/flow)
+- [Terminal Handbook](http://linuxcommand.org/)
+- [Command Prompt Handbook](http://www.makeuseof.com/tag/a-beginners-guide-to-the-windows-command-line/)
+- [Chat with us!](https://gitter.im/jboss-outreach)
```
Comment (aashutoshrathi) [inline README.md]: Remove Chat with us, we already have a gitter badge and also not suitable for subheading.

**Comment by HITGIF** [comment]
Done and squashed

**Pull request merged**